src/hooks/useUserLocation.js

```diff
@@ -16,11 +19,13 @@ export default function useUserLocation() {
     })
     const [ errorMessage, setErrorMessage ] = useState()
     const [ isCountyDataLoaded, setIsCountyDataLoaded] = useState(false)
-
+    
     useEffect(() => {userPlace && getCountyData()}, [userPlace])
     useEffect(() => {coordinates.longitude && getIncidenceData()}, [coordinates])
     useEffect(() => setErrorMessage(''), [userPlace,coordinates] )
```

Comment:
Could you combine line 23 and 25 in one useEffect?

Reply:
Sadly not, cause line 23 should just happen if there is a change with userPlace, but the error message should also be reset after coordinates changed